openquakeplatform/openquakeplatform/test/platform.py

```diff
@@ -0,0 +1,213 @@
+import time
+from utils import *
+
+class Platform(object):
+    DT = 0.1
+    def __init__(self):
+        self.driver   = None
+        self.basepath = None
+        self.user     = None
+        self.passwd   = None
+        self.debugger = None
+
+    def init(self):
+        try:
+            from openquakeplatform.test.config import pla_basepath, pla_user, pla_passwd, pla_debugger
+        except ImportError:
+            print "ERROR: config.py not found. Copy config.py.tmpl in config.py and modify it properly."
+            sys.exit(1)
+
```

Comment:
Better sys.exit("ERROR: config.py not found. Copy config.py.tmpl in config.py and modify it properly."): the error will be printed on stderr and not stdout.